electron-builder.json

```diff
@@ -18,7 +18,7 @@
     ]
   },
   "linux": {
-    "category": "Audio",
+    "category": "Audio;AudioVideo",
```

Comment:
@xyloflake But we have video player.

Reply:
@xyloflake On Discord.

Reply:
Notes for Audio category in [specifications](https://specifications.freedesktop.org/menu-spec/latest/apa.html#main-category-registry) states that

> Desktop entry must include AudioVideo as well

Otherwise it does not go to `Multimedia` tab on KDE, but to the `Lost & Found` tab.